create-new-data-pr.py

```diff
@@ -203,3 +203,7 @@ def get_tag_from_string(tag_string=None):
     change_tag_pull_request = dist_repo.create_pull(
         title=title, body=body, base=default_cms_dist_branch, head=repo_tag_pr_branch
     )
+
+    if data_pr_base_branch == data_repo_default_branch:
+        print("cms-data PR was for default branch, will merge now")
```

Comment:
@iarspider , may be add a comment to PR that why it is beening automatically merged.
Also I would suggest to a add a new command-line option e.g. `--no-merge`  (default value `False`) so we can use it (if needed) to only create the PR without merge